pkgs/applications/virtualization/rkt/default.nix

```diff
@@ -22,7 +22,7 @@ in stdenv.mkDerivation rec {
 
   stage1BaseImage = fetchurl {
     url = "http://stable.release.core-os.net/amd64-usr/${coreosImageRelease}/coreos_production_pxe_image.cpio.gz";
```

Comment:
Wait, scratch that LGTM. I replaced `alpha` with `stable` when I switched the version, so it should be switched back here.

This 404's:
```
curl -v -X HEAD http://stable.release.core-os.net/amd64-usr/794.1.0/coreos_production_pxe_image.cpio.gz
```